.github/workflows/pre-commit.yml

```diff
@@ -26,5 +26,14 @@ jobs:
           path: ~/.cache/pre-commit
           key: pre-commit|${{ steps.setup-python.outputs.python-version }}|${{ hashFiles('.pre-commit-config.yaml') }}
 
+      - name: Install hadolint
+        run: |
+          set -x
+          echo $PATH
+          wget -qO /usr/local/bin/hadolint https://github.com/hadolint/hadolint/releases/latest/download/hadolint-Linux-x86_64
```

Comment:
Not a veto, but considering that you normally enforce strict versioning for everything: Is the usage of `latest` intentional here?

Reply:
In this case, since the workflow has no write permissions and we do not care about the exact version of the tool used (at least for now), I decided that I'm fine with it.